hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/DBCheckpointServlet.java

```diff
@@ -229,9 +242,7 @@ private void generateSnapshotCheckpoint(HttpServletRequest request,
       long duration = Duration.between(start, end).toMillis();
       LOG.info("Time taken to write the checkpoint to response output " +
           "stream: {} milliseconds", duration);
-
-      LOG.info("Excluded SST {} from the latest checkpoint.",
-          excludedSstList);
+      logSstFileList(excludedSstList, msg, 5);
```

Comment:
> message should be a parameter

It should be a parameter to pass different message each time.

- `Received list of {} SST files to be excluded{}: {}`
- `Excluded {} SST files from the latest checkpoint{}: {}`

Sorry if it was not clear.